core-models/src/main/scala/com/mesosphere/usi/core/models/faultdomain/AnyDomain.scala

```diff
@@ -0,0 +1,7 @@
+package com.mesosphere.usi.core.models.faultdomain
+import org.apache.mesos.v1.Protos
+
+object AnyDomain extends DomainFilter {
```

Comment:
This would be used by the Jenkins plugin. See eg https://github.com/jenkinsci/mesos-plugin/pull/370